server/models/UserChapterRole.ts

```diff
@@ -23,15 +24,25 @@ export class UserChapterRole {
   @JoinColumn({ name: 'chapter_id' })
   chapter!: Chapter;
 
+  /*
+    This indicates whether he use wants to receive notifications about this chapter.
```

Comment:
Maybe rename this to notifications?
This sounds a lot like someone wanting to join a chapter

Reply:
Honestly this is fine as is, we can change it later if others find it confusing